src/Microsoft.ML.Data/TrainCatalog.cs

```diff
@@ -367,6 +368,20 @@ internal BinaryClassificationTrainers(BinaryClassificationCatalog catalog)
             return result.Select(x => new CrossValidationResult<CalibratedBinaryClassificationMetrics>(x.Model,
                 Evaluate(x.Scores, labelColumn), x.Scores, x.Fold)).ToArray();
         }
+
+        /// <summary>
+        /// The list of trainers for performing binary classification.
+        /// </summary>
+        public CalibratorsSubCatalog Calibrators { get; }
+
+        public sealed class CalibratorsSubCatalog : CatalogInstantiatorBase
```

Comment:
>CalibratorsSubCatalog [](start = 28, length = 21)

If I look at other classes derived from `CatalogInstantiatorBase`, I don't see anything like `SubCatalog`. How about just `CalibratorsCatalog`? Why do we need the sub in there?

Don't forget to like and sub.